scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -4147,10 +4147,7 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     val isIndented = acceptOpt[Indentation.Indent]
 
     while (!token.is[StatSeqEnd]) {
-      def isDefinedInEllipsis = {
-        if (token.is[LeftParen] || token.is[LeftBrace]) next(); statpf.isDefinedAt(token)
-      }
-      if (statpf.isDefinedAt(token) || (token.is[Ellipsis] && ahead(isDefinedInEllipsis)))
```

Comment:
Do we have a test case for it? When does the Ellipsis token is actually used?

Reply:
this code is from 5-6 years ago. it's possible that this execution path was never executed. also, it's possible that whatever `statpf` that is being passed here also takes care of ellipses.

Reply:
tracked down all partial functions passed here:
- `consumeStat`
- `templateStat`
- `topStat`

They all handle Ellipsis.